apps/settings/js/panels/simpin/simpin.js

```diff
@@ -89,14 +90,14 @@ define(function(require) {
 
       var simPinCheckbox =
         this._elements.simPinContainer.querySelector(
-          '.simpin-enabled-' + cardIndex + ' input');
+          '.simpin-enabled-' + cardIndex + ' gaia-switch');
 
       var isSimAvailable = icc && icc.cardState && icc.cardState !== 'unknown';
 
       // when fugu is in airplane mode, icc.cardState will not be changed ...
       // in this way, we have to use isAirplaneMode to check this situation
       if (!isSimAvailable || this.isAirplaneMode) {
-        simPinCheckbox.disabled = true;
+        simPinCheckbox.setAttribute('disabled', true);
```

Comment:
seems it's frequently used, why we do not use `Object.defineProperty( proto, 'disabled' ...` to define this property in `gaia-switch` element?